### PR TITLE
V1.8.1 event subscription

### PR DESCRIPTION
**Pull request by supritat**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

### CODE REVIEW

File: core/jazz_create-serverless-service/index.js [hunk 1]
```diff
@@ -268,28 +268,24 @@ var getServiceData = (service_creation_data, authToken, configData) => {
...
-                    }
 
                 } else {
                     logger.error('cronExpValidator : ', cronExpValidator);
                     reject(cronExpValidator);
                 }
             }
+            if (service_creation_data.events && service_creation_data.events.length) {
+              //Process events into properties
+              for (var idx = 0; idx < service_creation_data.events.length; idx++) {
```
Comment (svidhani) [inline core/jazz_create-serverless-service/index.js]: let?

Comment (svidhani) [reply core/jazz_create-serverless-service/index.js]: any update to unit tests (eg. sqs? )

Comment (supritat) [reply core/jazz_create-serverless-service/index.js]: Updated

**Comment by supritat** [comment]
Created a followup PR for event subscription changes in create-serverless-service api.
New PR: [https://github.com/tmobile/jazz/pull/509]

**Pull request closed without merging**